sd_meh/merge_methods.py

```diff
@@ -55,3 +57,15 @@ def triple_sum(
     a: Tensor, b: Tensor, c: Tensor, alpha: float, beta: float, **kwargs
 ) -> Tensor:
     return (1 - alpha - beta) * a + alpha * b + beta * c
+
+
+def multiply_difference(
+    a: Tensor, b: Tensor, c: Tensor, alpha: float, **kwargs
+) -> Tensor:
+    difference = torch.abs((a - c) * (b - c))
+    try:
+        difference = torch.sqrt(difference)
+    except RuntimeError:
+        difference = torch.sqrt(difference.float()).half()
```

Comment:
For some reason, on my CPU and torch version at least (was latest last week AFAIK), sqrt isn't implemented for half. It raises this error type so that's the bottom-up fix I found. Let me know if you have insight or a better fix.

Reply:
I believe this might be related: https://github.com/ultralytics/yolov5/issues/830

Reply:
You can see that it works if you use the same model for `a` and `b`: `(a - c) * (a - c) == (a - c)**2 == a - c`